lib/utils/__tests__/isStandardSyntaxAtRule.test.js

```diff
@@ -1,136 +1,112 @@
 'use strict';
 
 const isStandardSyntaxAtRule = require('../isStandardSyntaxAtRule');
-const less = require('postcss-less');
 const postcss = require('postcss');
-const sass = require('postcss-sass');
-const scss = require('postcss-scss');
+const postcssLess = require('postcss-less');
+const postcssSass = require('postcss-sass');
+const postcssScss = require('postcss-scss');
```

Comment:
[note] I've renamed the variables not to hide local variables with the same name, e.g.

```js
const sass = '@mixin mixin()\n  @content';
const scss = '@mixin mixin() { @content; };';
```